opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/ESamplingSurvivorGeneration.java

```diff
@@ -0,0 +1,34 @@
+package org.opt4j.optimizers.ea.aeseh;
+
+import java.util.Collection;
+import java.util.Set;
+
+import org.opt4j.core.Individual;
+
+import com.google.inject.ImplementedBy;
+
+/**
+ * 
+ * The {@link ESamplingSurvivorGeneration} generates the survivor pool during
+ * the selection implemented by {@link AeSeHSelector}.
+ * 
+ * @author Fedor Smirnov
+ *
+ */
+@ImplementedBy(DefaultSurvivorGeneration.class)
+public interface ESamplingSurvivorGeneration {
```

Comment:
Is it probably that we get another Survivor Generation? Otherwise, we could remove the interface in favor for just a single class.

If not, I would nonetheless vote for renaming this interface to SurvivorGeneration  and the DefaultSurvivorGeneration to EpsilonSamplingSurvivorGeneration.

Reply:
See https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it